code/client/src/core/modules/human.cpp

```diff
@@ -226,6 +214,7 @@ namespace MafiaMP::Core::Modules {
             return;
         }
 
+        // TODO: Explain why do we have to do this
         trackingData->human->GetHumanScript()->SetDemigod(true);
         trackingData->human->GetHumanScript()->SetInvulnerabilityByScript(true);
 
```

Comment:
Any ideas why?

Reply:
To ensure remote human does not die on client-side due to client-only factors, this way we mirror actual health the player has and don't run into desync issue where player would die on someone else's screen even if they shouldn't

Reply:
Absolutely